package.json

```diff
@@ -9,62 +9,37 @@
   "title": "WooCommerce Square",
   "version": "4.7.0",
   "homepage": "https://woocommerce.com/products/woocommerce-square/",
-  "main": "Gruntfile.js",
   "scripts": {
-    "build": "composer install --no-dev && NODE_ENV=production grunt && NODE_ENV=production npm run build:webpack && npm run archive",
+    "build": "composer install --no-dev && npm run build:webpack && npm run archive",
```

Comment:
In testing this command, the final zip that is built contains a `assets` directory and `src` directory. Are either of these needed for the final plugin? If not, we should exclude those from the archive

Reply:
Not sure why the `assets` directory still shows after switching to this branch. But if you delete it, it doesn't show in the git diff.